---
# How many times per second should the system be queried for stats
update_frequency: 2

# These settings are quick_switches to completely disable certain parts of statistic collection
do_cpu: true
do_memory: true
do_network: true
